hook.py

```diff
@@ -8,10 +8,14 @@
 
 
 async def enable(services):
-    Popen(['haproxy', '-q', '-f', 'plugins/ssl/templates/haproxy.conf'])
+    haproxy_conf = Path('plugins/ssl/templates/haproxy.conf')
+    user_conf = Path('plugins/ssl/conf/haproxy.conf')
+    if user_conf.is_file():
+        haproxy_conf = user_conf
+    Popen(['haproxy', '-q', '-f', haproxy_conf])
     logging.debug('Serving at https://127.0.0.1:8443')
```

Comment:
Oh, can you correct this line too? Our haproxy config has us listening on 0.0.0.0 not just 127.0.0.1 https://github.com/mitre/ssl/blob/master/templates/haproxy.conf#L17